docs/docs/references/users/README.md

```diff
@@ -189,27 +188,27 @@ Sample _custom data_ from an admin user in Logto:
 }
 ```
 
-Each user's _custom data_ is stored in an individual JSON object.
+Each user's custom_data is stored in an individual JSON object.
 
-:::caution Put sensitive data in _custom data_
+:::caution Put sensitive data in custom_data
```

Comment:
```suggestion
:::caution DO NOT put sensitive data in custom_data
```

Reply:
this fix is put in next PR. please ignore